modules/nw-ovn-kubernetes-matrix.adoc

```diff
@@ -6,65 +6,64 @@
 [id="nw-ovn-kubernetes-matrix_{context}"]
 = Supported network plugin feature matrix
 
-{openshift-networking} offers two options for the network plugin, OpenShift SDN and OVN-Kubernetes, for the network plugin. The following table summarizes the current feature support for both network plugins:
+{openshift-networking} offers OVN-Kubernetes as the network plugin. The following table summarizes the current feature support for both network plugins:
```

Comment:
```suggestion
{openshift-networking} offers OVN-Kubernetes as the network plugin. The following table summarizes the current feature support:
```

Though probably this table is not useful any more anyway; it's like a subset of the table of contents except without links...

Reply:
@danwinship I don't disagree. I'm going to move your suggestion into a ContentX ticket and will try to get to it after this merges. Thank you.